requirements.txt

```diff
@@ -24,12 +24,12 @@ dealer
 flask-collect
 flake8
 passlib
-twill
+https://github.com/alexandre/twill/tarball/master
```

Comment:
https://github.com/quokkaproject/twill (also your are added to the organisation)